src/nvim/version.c

```diff
@@ -187,7 +187,7 @@ static const int included_patches[] = {
   2297,
   2296,
   2295,
-  2294,
+  // 2294,
```

Comment:
Due to the partial reverts.
But, was the intended fix ported though or was it N/A in the end?